src/components/cylc/gscan/GScan.vue

```diff
@@ -428,14 +430,6 @@ export default {
       this.isLoading = !isActive
     },
 
-    getWorkflowIcon (status) {
-      let wstatus = WorkflowState.enumValueOf(status.toUpperCase())
```

Comment:
Perhaps we should (not now) override `enumValueOf` to make this case insensitive and provide the error fallback?

Reply:
Sounds good to me!